manager/src/main/scala/org/labrad/registry/Registry.scala

```diff
@@ -235,17 +235,48 @@ extends LocalServer with Logging {
     def getValue(key: String, pat: String, set: Boolean, default: Data): Data = _getValue(key, pat, Some((set, default)))
 
     def _getValue(key: String, pat: String = "?", default: Option[(Boolean, Data)] = None): Data = {
-      val data = store.getValue(curDir, key, default)
+      val (data, textOpt) = store.getValue(curDir, key, default)
       val pattern = Pattern(pat)
       data.convertTo(pattern)
     }
 
+    @Setting(id=21,
```

Comment:
I think this setting needs some docs explaining how it works. In our discussion yesterday it took a while before it came out that the string can only be stored if it can be parsed to LabRAD data. This might seem obvious, but it's an important feature. It's probably also worth explicitly noting that the point of this setting is to use the user as a pretty-printer and that really only editors (i.e. scalabrad-web) should use it.